docs/tutorials/ratelimit.md

```diff
@@ -152,7 +161,16 @@ clusterClientRatelimit("groupB", 10, "1m")
 The same for Authorization Header you would use:
 
 ```
-clusterClientRatelimit("groupC", 10, "1m", "auth")
+clusterClientRatelimit("groupC", 10, "1m", "Authorization)
+```
+
+The optional fourth argument can create an AND combined Header
+ratelimit, if you use `,` to separate the them. For example all of the
```

Comment:
"ratelimit, if you use , to separate the them" -> "ratelimit. The headers names must be separated by ,."

Reply:
fixed